dagfactory/_yaml.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
+
+import os
+
+import yaml
+
+from .utils import cast_with_type
+
+
+def load_yaml_file(file_path: str) -> dict[str, any]:
```

Comment:
probably could also make this method private

Reply:
Addressed in 9caa6b8fddf2db8c397b1053d6db3f5018a5728d